optuna/storages/_base.py

```diff
@@ -54,25 +54,6 @@ class BaseStorage(object, metaclass=abc.ABCMeta):
     any successive reads on data `X` from the same process must read the written
     value or one of the more recent values.
 
-    **Stronger consistency requirements for special data**
```

Comment:
Where should I put this comment? Can it be moved to `storages._cached_storage.py`?

Reply:
I'm not sure, but I think we don't have to move this comment, since the BaseStorage and _CachedStorage are not public. We should consider the appropriate place of this comment when we make them public.

Reply:
Ok. I reverted it.